contracts/Colony.sol

```diff
@@ -313,8 +312,13 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     );
   }
 
+  // v3 to v4
   function finishUpgrade() public always {
-    // Nothing here for v2 to v3, but it needs to be defined.
+    ColonyAuthority colonyAuthority = ColonyAuthority(address(authority));
+
+    // Remove function
+    bytes4 sig1 = bytes4(keccak256("setPaymentDomain(uint256,uint256,uint256,uint256)"));
```

Comment:
This could be a constant rather than being calculated on-chain during the upgrade.